### ISSUE TITLE
Duplicate module name

### ISSUE THREAD

**Issue by sfratini** [CleverTap/clevertap-react-native#7]
Hi! We are integratin this package and it says that the module is duplicated as it is compiling RN twice. We have a fixed RN version and this module is always trying to install a newer version in the private depdendencies. 

```
 Duplicate module name: react-native
  Paths: <package>\Examples\node_modules\clevertap-react-native\node_modules\react-native\package.json collides with <package>\Examples\node_modules\react-native\package.json
```

A workaround is to rename the clevertap-react-native package.json to the exact version we have which is 0.48.1 but it is a pain to do it every time. Specially because we use a build machine. 

Thanks!

**Comment by pwilkniss** [comment]
thanks @sfratini   this should be fixed in v0.1.8
https://github.com/CleverTap/clevertap-react-native/commit/0925f538b6820ed1c922355a10ac6fa86b0d5a1e

**Comment by sfratini** [comment]
I can confirm this is fixed in 0.1.8. Thanks!
